src/Spryker/Glue/AppKernel/Builder/ResponseBuilder.php

```diff
@@ -40,7 +40,7 @@ public function buildRequestNotValidResponse(
 
             $errors[] = [
                 'code' => $error->getCode(),
-                'message' => $error->getMessage(),
+                'detail' => $error->getMessage(),
```

Comment:
I refactored it last week to be message instead of detail as the GlueErrorTransfer also uses message and not detail. Is it wrong on the Registry side? If it is wrong on the Registry side I would prefer to fix it there and check for both values until the next major release.

Reply:
Agree, I will create PR for the Registry

Reply:
https://github.com/spryker-projects/registry-service/pull/587